### PR TITLE
3.1.0 history

### PR DESCRIPTION
**Pull request by jburel**
Prep changelog ahead of new release

cc @will-moore

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,6 +18,10 @@ Requirements
 
 * OMERO 5.3.0 or newer.
 
+To able to export the figure with italic or bold label, it is necessary
```
Comment (will-moore) [inline README.rst]: This is just for TIFF export. PDF export doesn't need those fonts.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
+3.1.0 (June 2017)
+-----------------
+
+- Add support for markdown in panel labels (PR #209)
```
Comment (will-moore) [inline CHANGELOG.md]: "Add support for italics and bold in panel labels using markdown syntax" might be nicer for users not familiar with markdown.

**Comment by jburel** [comment]
@will-moore text updated

**Comment by will-moore** [comment]
Changes look good, thanks.

**Pull request merged**